### DISCUSSION TITLE
Inline typst code

### DISCUSSION THREAD

**Discussion by psads-git** [quarto-dev/quarto-cli#8870] [Q&A] [answered]
Consider the following `quarto` document, where I try to use `typst` inline code (to avoid inserting inline mathematical formulas with `LaTeX` code), but it does not work. I guess that is not implemented. Am I right? Thanks!

```
---
format: typst
---

To use typst code inline, I have tried `{typst} $integral x dif x$` and 
also `{=typst} $integral x dif x$`.

```

**Comment by mcanouil** [comment, marked as answer]
Typst is not a computing language.
Raw code syntax are as follows:
- inline: `` `whatever`{=FORMAT} ``
- multiline: ` ```{=FORMAT} `

**Reply by psads-git** [reply]
Thanks, @mcanouil: that is it!

**Reply by ErickFG08** [reply]
Thanks @mcanouil! I was also searching for this!
